wrappers/c/zxing.cpp

```diff
@@ -0,0 +1,159 @@
+#include "zxing.h"
+
+#include <ZXing/ImageView.h>
+#include <ZXing/BarcodeFormat.h>
+#include <ZXing/DecodeHints.h>
+#include <ZXing/Result.h>
+#include <ZXing/ReadBarcode.h>
+
+extern "C" {
+
+/*
+ * ZXing/ImageView.h
+ */
+
+zxing_ImageView * zxing_ImageView_new (const uint8_t * data, int width, int height, zxing_ImageFormat format, int rowStride, int pixStride)
```

Comment:
To fit the style of the rest of the project, would you please remove the space between the typename and the '*'? Or maybe simply let clang-format with the config in the top-level directory take care of that`